Dockerfile

```diff
@@ -1,27 +0,0 @@
-FROM ghcr.io/graalvm/native-image:ol9-java17-22.3.3 AS builder
```

Comment:
Is there some new support for native images in Spring Boot 3.2.x, or why don't we need this anymore?

Reply:
Sadly, no. Since there is no such image yet for Java 21, the only easy way to produce a native image is to use buildpacks. We can think about pushing non-native ARM images though.